docs/new-audits.md

```diff
@@ -0,0 +1,30 @@
+So, you want to create a new audit? Great! We're excited that you want to add to the Lighthouse project :) The goal of this 
+document is to help you understand what constitutes as a "good" audit for Lighthouse, and steps you can follow if you want
+to propose a new audit. 
+
+## New audit principles
+Lighthouse audits that surface in the report should be: 
+- Applicable to a significant portion of web developers (based on scale and severity of impact) 
+- Contribute significantly towards making the mobile web experience better for end users. 
+- Not have a significant impact on our runtime performance or bundle size. 
+- Something that is new, and not something that is already measured by existing audits. 
+- Important for our strategic goals as a product (i.e. could other Lightrider customers benefit from this? What about other GBO teams?)
+- Measurable (especially for performance audits) or have clear pass/fail states. 
+- Not use 3rd party APIs for completing the audit check. 
+
+*Note: it's possible to submit a new audit proposal for something that you as a contributor feel very passionate about, but the 
+Lighthouse team feels that it wouldn't add value to a significant portion of the Lighthouse users. In this case, we'd be open to 
+having your audit be a part of the report when run with the full config on the Command Line. * 
+
+## Process for creating a new audit
+- Briefly scan the criteria we’ve laid out above. If you think the principles match with your proposed new audit, then proceed! 
```

Comment:
lets use numbered bullets for this one instead ?  you can just use `1.` every time (and it'll increment)

Reply:
done!